kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiConnection.java

```diff
@@ -164,7 +172,7 @@ public KyuubiConnection(String uri, Properties info) throws SQLException {
 
       // open client session
       openSession();
-      getLaunchEngineLog();
+      showLaunchEngineLog();
```

Comment:
The kyuubi jdbc driver should not intent to support `EmbeddedMode`?

If yes, can we check `isEmbeddedMode`'s value and throw exception if it is `true`?

Reply:
i think it is better to reserve this.

Reply:
it does not matter.